notebooks/sampling/RandomWalk.ipynb

```diff
@@ -0,0 +1,223 @@
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Read The Data\n",
+    "# Define the path to the test data  \n",
+    "datafile='../data/karate-data.csv'\n",
+    "\n",
+    "gdf = cudf.read_csv(datafile, delimiter='\\t', names=['src', 'dst'], dtype=['int32', 'int32'] )"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 3,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "gdf['wt'] = 1.0"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 4,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Create a Graph - using the source (src) and destination (dst) vertex pairs from the Dataframe \n",
+    "G = cugraph.Graph()\n",
+    "G.from_cudf_edgelist(gdf, source='src', destination='dst', edge_attr='wt')"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 5,
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "(34, 78)"
+      ]
+     },
+     "execution_count": 5,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "# some stats on the graph\n",
+    "(G.number_of_nodes(), G.number_of_edges() )"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 6,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# create a list with the seeds\n",
+    "seeds = [17,19]"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 7,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "rw, os = cugraph.random_walks(G, seeds, 4)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "the function return two results, let's look at the second one first.  The seed offset, which is a cuDF Series,\n",
+    "is orders based on the order of the seeds and provides an offset into the results dataframe\n",
+    "of where the path for that seed starts.  The series is one larger than the number of seeds, so it is easy to compute the number of rows that make up the path.  We need to know the number of rows since that could be different than the max path length."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 8,
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "0    0\n",
+       "1    3\n",
+       "2    6\n",
+       "dtype: int64"
+      ]
+     },
+     "execution_count": 8,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "os"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 9,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "seed 17 starts at index 0 and is 3 rows\n",
+      "seed 19 starts at index 3 and is 3 rows\n"
+     ]
+    }
+   ],
+   "source": [
+    "for i in range(len(seeds)):\n",
+    "    print(f\"seed {seeds[i]} starts at index {os[i]} and is {os[1 + 1] - os[1]} rows\")"
+   ]
+  },
+  {
```

Comment:
You don't describe or show the values in rw.  Should probably display the paths for each of the seeds with a block of notebook code here.

Reply:
oops = got pulled off on to another task and forgot to finish this.

Reply:
fixed